runner/src/main.rs

```diff
@@ -331,6 +331,7 @@ struct Example {
 enum Target {
     Bazel { bazel_target: String },
     Cargo { cargo_manifest: String },
+    NPM { package_directory: String },
```

Comment:
I don't think Rust has a clear style preference for initialisms, but I think `Npm` would be more in line with enum naming conventions here (all uppercase is usually reserved for constants). Do use the correct capitalisation in strings and log entries though.